lib/public_key/src/pubkey_os_cacerts.erl

```diff
@@ -216,3 +224,35 @@ load_nif() ->
             end;
         Error1 -> Error1
     end.
+
+%%%
+%%% Error Handling
+%%%
+
+conv_error_reason(enoent) -> enoent;
+conv_error_reason({enotsup, _OS}) -> enotsup;
+conv_error_reason({eopnotsupp, _Reason}) -> eopnotsupp;
+conv_error_reason({eopnotsupp, _Status, _Acc}) -> eopnotsupp.
+
+-spec format_error(Reason, StackTrace) -> ErrorMap when
```

Comment:
So I guess the intention is that public_key:cacerts_load should call this function?

Reply:
So I think I might have initially misunderstood.  I take this back, it is an other discussion!

Reply:
@IngelaAndin I've updated the PR description to include a description of intention / changes. Does this answer your question?